scripts/kubecf-wait.sh

```diff
@@ -63,3 +63,25 @@ mapfile -t endpoints < <(get_resource endpoints)
 for endpoint in "${endpoints[@]}" ; do
     RETRIES=180 DELAY=10 retry wait_for_endpoint "${endpoint}"
 done
+
+green "Waiting for all services to have associated external IPs"
+wait_for_service_external_ips() {
+    selector=(
+        'app.kubernetes.io/component=router'
+        '!quarks.cloudfoundry.org/instance-group-name'
+    )
+    jsonpath='{.items[].status.loadBalancer.ingress[].ip}'
```

Comment:
Wouldn't this only make sense for LB configurations? (It wouldn't make sense for minikube, for example.)

Reply:
True. I gave it a try, but I suppose this check will end up just being completely skipped in the other cases for now.

Reply:
Decided to just move the wait-for-ip code into a script in pull-request-ci/, for now.